src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const callbackNull = (a, b) => {
```

Comment:
`callbackNull` looks like strange name :)
It's better to name it smth like `sortDefault`.